src/Elements/TriangularFace.jl

```diff
@@ -0,0 +1,27 @@
+using ..Elements: AbstractFace, AbstractNode
+using ..Utils: eye
+
+export TriangularFace
+
+"""
+A `TriangularFace` represents an element composed by three `Node`s.
+### Fields:
+- `n₁`             -- stores first triangle node.
+- `n₂`             -- stores second triangle node.
+- `n₂`             -- stores third triangle node.
+- `label`          -- stores the triangle label.
+"""
+struct TriangularFace{dim,N<:AbstractNode{dim},T<:Real} <: AbstractFace{dim,T}
+    n₁::N
+    n₂::N
+    n₃::N
+    label::Symbol
+    function TriangularFace(n₁::N, n₂::N, n₃::N, label=:no_labelled_face) where
+    {dim,T<:Real,N<:AbstractNode{dim,T}}
+        new{dim,N,T}(n₁, n₂, n₃, Symbol(label))
+    end
+end
+
+"Returns a `Vector` of`Node`s for the `TriangularFace` `tf`."
+nodes(tf::TriangularFace) = [tf.n₁, tf.n₂, tf.n₃]
```

Comment:
this allocates; you may want to revisit later, storing an SVector directly.

Reply:
Okay, I also considered to store the nodes of the elements into a Vector instead of separated files:

```julia
struct TriangularFace{dim,N<:AbstractNode{dim},T<:Real} <: AbstractFace{dim,T}
    nodes::SVector{N}
    label::Symbol
    function TriangularFace(n₁::N, n₂::N, n₃::N, label=:no_labelled_face) where
    {dim,T<:Real,N<:AbstractNode{dim,T}}
        new{dim,N,T}(SVector(n₁, n₂, n₃), Symbol(label))
    end
end

"Returns the nodes for the `TriangularFace` `tf`."
nodes(tf::TriangularFace) = [tf.n₁, tf.n₂, tf.n₃]
```

Reply:
Open this in this issue #80